.buildkite/pipeline.yml

```diff
@@ -185,6 +185,7 @@ steps:
           - "--app=build/outputs/apk/release/app-release.apk"
           - "--farm=bs"
           - "--device=ANDROID_7_1"
+          - "--fail-fast"
```

Comment:
I wasn't sure on the justification for adding these fast-fails – is it because we have a retry mechanism, so it should have been on fast fail before?

Reply:
It's just to get feedback of a failing Buildkite step as soon as possible.  If the MR tests fail it's generally because we've forgotten something obvious and there's no point running the rest of the tests.  It's really annoying when you realise a pipeline has been failing for several minutes before you get any feedback.

One of the main motivations was that the BitBar test, for now, use the secure tunnel and can often have bad runs.  I want them to fail asap so I can retry in that case.  (We can/will move them not to use the secure tunnel, I just wanted something in place quickly and not have to refactor the test fixture).